src/sentry/static/sentry/app/components/issueDiff.jsx

```diff
@@ -0,0 +1,102 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import ApiMixin from '../mixins/apiMixin';
+import LoadingIndicator from './loadingIndicator';
+import rawStacktraceContent from './events/interfaces/rawStacktraceContent';
+
+import '../../less/components/issueDiff.less';
+const IssueDiff = React.createClass({
+  propTypes: {
+    baseIssueId: PropTypes.string.isRequired,
+    targetIssueId: PropTypes.string.isRequired,
+    baseEventId: PropTypes.string.isRequired,
+    targetEventId: PropTypes.string.isRequired
+  },
+
+  mixins: [ApiMixin],
+
+  getDefaultProps() {
+    return {
+      baseEventId: 'latest',
+      targetEventId: 'latest'
+    };
+  },
+
+  getInitialState() {
+    return {
+      loading: true,
+      baseEvent: {},
+      targetEvent: {},
+      Diff: null
+    };
+  },
+
+  componentWillMount() {
```

Comment:
I've been consistently using `componentDidMount` for fetching (instead of `componentWillMount`). Can we standardize on one?

My reason for `componentDidMount`: I think it signals an acknowledgement that the component will definitely render/mount before the fetches finish. But admittedly, this is like ... a pretty minor thing.

Thoughts?

Reply:
`componentDidMount` is the recommended lifecycle method to perform API requests, so I would vote that. 

Looks like we're about split in the codebase right now, but should def start standardizing this in future PRs.
```
❯ ag componentWillMount | wc -l
      68
❯ ag componentDidMount | wc -l
      42
```